public/redux/actions/appStateActions.js

```diff
@@ -180,3 +180,15 @@ export const updateWithUserLogged = (withUserLogged) => {
    withUserLogged,
   };
 };
+
+
+/**
+ * Updates allowedAgents in the appState store
+ * @param GET_ALLOWED_AGENTS
```

Comment:
The name of  the `@param` doesn't match with the declaration of the function.

Reply:
Thanks for reporting it, it has been patched